grapher/AboutBox.resx

```diff
@@ -0,0 +1,225 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not a fan of having loose resx files around unless they are actually used. Did you add this on purpose?

Reply:
this one was added by automatically by VS template